pymeasure/instruments/srs/sr830.py

```diff
@@ -582,6 +639,30 @@ def get_buffer(self, channel=1, start=0, end=None):
         return self.binary_values("TRCB?%d,%d,%d" % (
             channel, start, end - start))
 
+    def get_buffer_bytes(self, channel=1, start=0, end=None):
+        """ Acquires the 32 bit floating point data through bytes transfer
+        This is the fastest transfer method according to the manual.
+        The byte to float conversion is provided in the manual
+        and implemented in :meth:`buffer_bytes_convert`
+        """
```

Comment:
According to PEP257, you should have a single summary line, then an empty line and then a more detailed explanation.
All should be in imperative mode (Acquire instead of Acquires...)